core/src/main/java/org/mskcc/cbio/oncokb/util/CacheUtils.java

```diff
@@ -222,6 +244,25 @@ private static void notifyOtherServices(String cmd, Set<Integer> entrezGeneIds)
             System.out.println("Register other services: " + MainUtils.getTimestampDiff(current) + " at " + MainUtils.getCurrentTime());
             current = MainUtils.getCurrentTimestamp();
 
+            final String s3AccessKey = PropertiesUtils.getProperties("aws.s3.accessKey");
+            final String s3SecretKey = PropertiesUtils.getProperties("aws.s3.secretKey");
+            final String s3Region = PropertiesUtils.getProperties("aws.s3.region");
```

Comment:
What happens when these keys are not available? Will there be exceptions when initiating aws s3 client?

Reply:
Yes I will add a check to handle this